frontend/src/app/testQueue/QueueItem.tsx

```diff
@@ -749,7 +755,12 @@ const QueueItem = ({
   };
 
   function getDeviceTypeOptions() {
-    let deviceTypeOptions = [...facility!.deviceTypes]
+    let deviceTypes = facility!
+      .deviceTypes as DeviceWithoutModelOrManufacturer[];
```

Comment:
**question:** can we still clone the array with the spread operator or does the rsv filter function require it to be passed by reference?

Reply:
I think they'd be functionally equivalent. Is there a preference you have for one or the other?

Reply:
As long as everything still works as expected I'd prefer cloning it to make sure we don't accidentally modify the original array on `facility`, and we can still type cast it as `[...facility!.deviceTypes] as DeviceWithoutModelOrManufacturer[]`